pkg/api/rpms.go

```diff
@@ -92,6 +92,30 @@ type SearchRpmResponse struct {
 	Summary     string `json:"summary"`      // Summary of the package found
 }
 
+type SearchModuleStreamsRequest struct {
+	UUIDs    []string `json:"uuids" validate:"required"`     // List of repository UUIDs to search
+	RpmNames []string `json:"rpm_names" validate:"required"` // List of rpm names to search
+	SortBy   string   `json:"sort_by"`                       // SortBy sets the sort order of the result
+	Search   string   `json:"search"`                        // Search string to search rpm names
+	Offset   int      `json:"offset"`                        // Starting point for retrieving a subset of results.
+	Limit    *int     `json:"limit,omitempty"`               // Maximum number of records to return for the search
+}
+
+type Stream struct {
+	Name string `json:"name"` // Stream name
```

Comment:
I think 'name' here is the same name as the Module, you probably want to include 'stream' which is the version of the module (i.e. "2.5").  Other fields I'd probably include:  version, context, arch, summary, description

Reply:
I think you misunderstand, this is the stream version number, or name of the stream (2.5 for example).

Reply:
when i look at a module md document, the name is typically the name of the module:

```
document: modulemd
version: 2
data:
  name: ruby
  stream: 2.5
  version: 8030020200624105530
  context: 30b713e6
  arch: x86_64
  summary: An interpreter of object-oriented scripting language
  description: >-
    Ruby is the interpreted scripting language for quick and easy object-oriented programming.  It has many features to process text files and to do system management tasks (as in Perl).  It is simple, straight-forward, and extensible.
  license:
```

and when i look at the rpm_modulemd table in pulp, it looks similar?

Reply:
I did rewrite the description of the parent class to make this a bit understandable from looking at the structs.